pkgs/games/chromium-bsu/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchurl
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, fontconfig
+, freealut
+, freeglut
+, ftgl
+, gettext
+, libGL
+, libGLU
+, openal
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "chromium-bsu";
+  version = "0.9.16.1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/chromium-bsu/Chromium%20B.S.U.%20source%20code/${pname}-${version}.tar.gz";
+    hash = "sha256-ocFBo00ZpZYHroEWahmGTrjITPhrFVRi/tMabVbhYko=";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    pkg-config
+  ];
+  buildInputs = [
+    SDL2
+    SDL2_image
+    SDL2_mixer
+    fontconfig
+    freealut
+    freeglut
+    ftgl
+    libGL
+    libGLU
+    openal
+  ];
+
+  # Autodetection is somewhat buggy
+  configureFlags = [
+    "--disable-glpng"
+    "--disable-glc"
```

Comment:
GLC is provided by `quesoglc`, at least in Gentoo it is used by the package: https://github.com/gentoo/gentoo/blob/master/games-action/chromium-bsu/chromium-bsu-0.9.15.1-r1.ebuild#L19

Reply:
Done!